apps/server/src/modules/user-migration/service/user-migration.service.spec.ts

```diff
@@ -18,12 +24,29 @@ describe('UserMigrationService', () => {
 	let module: TestingModule;
 	let orm: MikroORM;
 	let service: UserMigrationService;
+	let logger: Logger;
 
 	let schoolService: DeepMocked<SchoolService>;
 	let systemService: DeepMocked<SystemService>;
+	let userService: DeepMocked<UserService>;
+	let accountService: DeepMocked<AccountService>;
+
+	const hostUri = 'http://this.de';
+	const apiUrl = 'http://mock.de';
 
 	beforeAll(async () => {
-		jest.spyOn(Configuration, 'get').mockReturnValue('http://this.de');
+		jest.spyOn(Configuration, 'get').mockImplementation((key: string): unknown => {
```

Comment:
you can instead overwrite the config, using
```
configBefore = Configuration.toObject({ plainSecrets: true }); // beforeAll
Configuration.set('HOST', hostUri); // wherever needed, beforeAll, setup, or even within a single test
...
Configuration.reset(configBefore); // afterAll
```

Reply:
done